src/components/Calculator.js

```diff
@@ -1,35 +1,49 @@
 import { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      operation: null,
+      next: null,
+      total: null,
+    };
+  }
+
+  Calculation(event) {
+    this.calcButtons = event.target.innerText;
```

Comment:
[OPTIONAL] Class methods are written in camelCase, it’s a best practice to actually tell what the function is doing by giving the function name a verb as prefix.
```js

// bad
name(firstName, lastName) {
  return `${firstName} ${lastName}`;
}

// bad
Name(firstName, lastName) {
  return `${firstName} ${lastName}`;
}
 
// good
getName(firstName, lastName) {
  return `${firstName} ${lastName}`;
}
```

Reply:
Thank you for this, I would update the method declaration 👍